owiddata/generate-owiddata-stats.py

```diff
@@ -45,6 +73,52 @@ def convert_date(git_date):
     # Assumes the year will not begin with 0
     return datetime.datetime.fromisoformat(git_date).strftime('%B %d, %Y').replace(' 0', ' ')
 
+def retrieve_platform_types():
+    """Use trackvaccines.org to scrape the website listing approved vaccines
+    Returns: dataframe """
+    vaccine_info = dict()
+    vaccineHTML = urllib.request.urlopen('https://covid19.trackvaccines.org/vaccines/approved/')
+
+    # Extract the HTML that makes the cards on the webpage (each card is a vax)
+    soup = BeautifulSoup(vaccineHTML, "html5lib")
+    body = soup.find('body')
+    cards = body.find_all('li')
+
+    # Iterate through the cards to extract information
+    # Tags were identified empirically and are not self-evident
+    for card in cards: # find all element of tag
+        if card.find('a', {"class": "icon-link"}) is not None:
+            vaccine_type = card.find('a', {"class": "icon-link"}).get_text()
+            if vaccine_type.upper() != vaccine_type: #DNA, RNA, VLP
+                vaccine_type = vaccine_type.lower()
+            vaccine_category = platforms(vaccine_type)
```

Comment:
What's the difference between a type and a category?

Reply:
I changed them to "platforms" and "platform types" to try to clarify this, since I'm not totally sure on the correct terminology. So for example, plasmid vectored, replicating DNA, and non-replicating DNA would all be types of DNA vaccines